src/components/Editor/EditPublications.vue

```diff
@@ -329,24 +335,79 @@
             },
             openEditor: false,
             currentPublicationIndex: false,
-            citations_ids: []
+            citations_ids: [],
+            initialized: false,
+            initialFields: []
           }
         },
         computed: {
-          ...mapState('record', ['currentRecord', 'recordUpdate']),
+          // TODO: Remove
+          //...mapState('record', ['currentRecord', 'recordUpdate']),
           ...mapState('users', ['user']),
-          ...mapGetters("record", ["citations"]),
+          ...mapGetters("record", ["citations", "getSection", "getChanges", "currentRecord"]),
+          section(){
+            return this.getSection('publications');
+          },
+          message(){
+            let error = this.getSection("publications").error;
+            return {
+              error: error,
+              value: this.getSection("publications").message,
+              type: function(){if (error){return "error"} else {return "success"}}
+            };
+          },
+          publications: {
+            get() { return this.getSection("publications").data; },
+            set(newValue) {
+              this.$store.commit('record/setPublications', newValue);
+            }
+          },
+          metadata() {
+            return this.getSection("generalInformation").data.metadata
+          }
+
+        },
+        watch: {
+          publications: {
+            deep: true,
+            handler(newVal){
+              let _module = this;
+              /* istanbul ignore next */
+              if (!_module.initialized) return;
+
+              let updated = _module.checkChanges(newVal);
+              /*
```

Comment:
please use double * so the document generator understands to compile it. now it's known as comment

/**
* comment
*/

Reply:
This is merely a comment and is not intended for the documentation.